src/brasil/gov/tiles/upgrades/v4100/__init__.py

```diff
@@ -110,6 +110,29 @@ def migrate_deprecated_tiles(setup_tool):
     logger.info('Done')
 
 
+def migrate_tile_removed_override(setup_tool):
+    """Migrate list tile override:
```

Comment:
I would love to see a better description of the issue we are fixing with this upgrade step.